src/constants.ts

```diff
@@ -21,3 +23,8 @@ export enum Events {
 }
 
 export const RefKey = "GITHUB_REF";
+
+export const stateToOutputMap = new Map<string, string>([
```

Comment:
This can be moved to NullStateProvider as that is right owner/consumer for this map.

Reply:
Moved